examples/slack-connect-example/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Let's double check, but I don't think this is necessary because there's a gitignore in the root. If we need to, we can extend that one